examples/chip-tool/BUILD.gn

```diff
@@ -25,11 +25,25 @@ config("includes") {
 }
 
 chip_data_model("data_model") {
-  cluster_sources = [ "media-playback-client" ]
+  zap_file = "chip-tool.zap"
 
   zap_pregenerated_dir = "gen"
 
   use_default_client_callbacks = true
+
+  # These clusters are configured in the ZAP file but don't compile.
+  remove_cluster_sources = [
+    "color-control-server",
+    "door-lock-client",
+    "groups-client",
+    "groups-server",
+    "ias-zone-client",
+    "ias-zone-server",
+    "identify",
+    "level-control",
+    "scenes",
+    "scenes-client",
```

Comment:
All the "server" bits here look like bugs that don't make sense in this app?  @vivien-apple?

Or are we generating the client commands from the server command listing?